packages/calcite-design-tokens/support/token-transformer/format/javascript.ts

```diff
@@ -0,0 +1,22 @@
+import StyleDictionary, { Dictionary, File } from "style-dictionary";
+import { default as JsonToTS } from "json-to-ts";
+
+type FormatOptions = { dictionary: Dictionary; file: File };
+
+export function jsModule({ dictionary, file }: FormatOptions): string {
```

Comment:
Nitpick: for readability, could these be named `formatJs`/`formatTs` (or similar)?